lib/ex_docker_build/docker_build.ex

```diff
@@ -160,8 +160,12 @@ defmodule ExDockerBuild.DockerBuild do
     |> ExDockerBuild.create_layer()
   end
 
-  # defp exec({"ARG", args}, context, _path) do
-  # end
+  defp exec({"ARG", args}, context, _path) do
+    args = String.split(args, "=")
+
+    Map.merge(context, %{"Args" => Enum.at(args, 2)})
```

Comment:
`Args` attribute doesn't exist, doing a short search through google i didn't found anything related to this, so, don't even know if this is possible, did you test it?

Reply:
i found something https://docs.docker.com/engine/api/v1.37/#operation/ImageBuild, it has an attribute called `buildargs` which is only used when building the image, but not the container

Reply:
I didn't test using for real, only the test cover. I think that we had some args in the Bryan example.

Reply:
You can see an example of use [here](https://github.com/esl/packages-pipeline/blob/68597f05ce82c14787e1e13f40a0c5326ad4eb4d/erlang/ctx/Dockerfile_rpm#L2-L16)  - and [here](https://github.com/esl/packages-pipeline/blob/39921c127a66d83df277f32d8216c030f908da43/elixir/bin/build.sh#L19-L29)

Reply:
I haven’t been able to find anything about how to add support for `ARG`, I found about `buildargs` but is only supported when building a Dockerfile via `build` but we are not using it because we are building it manually (docs.docker.com/engine/api/v1.37/#operation/ImageBuild).